2024-06-24/minutes.md

```diff
@@ -1,3 +1,188 @@
+ 
+[Florian] Yeah, I can also add on that's 1 of the things I realized that required inconsistent. So when I read it the side bars. Actually, I'm adding more of the where we previously did not. At them and I need to figure out if you wanna don't wanna have them.
+
+[Ruth] Yeah, exactly.
+
+[Florian] And, and at the same time it's it's quite easy like there's a global post processing of all links where we have this context so just injecting them is like really nothing to do.
+
+[Ruth] That's what I'm trying to say on. Like I, I want people to vote. But no implementation to happen at this time, that's what I'm saying.
+
+[Florian] I was wondering about one case I ran into and that I was not aware of like there's guide pages that have like a feature status is that
+
+[Ruth] Yes, because they've got a BCD key. And we're using the BCD key for the baseline status.
+
+[Florian] But the feature is it called feature status, don't get me wrong.
+
+[Ruth] Okay. So yeah, I'm interested to see how maintainer feel about it. I think we also need to ask our users, but we can roll that into some decisions that we're gonna be making about the platform anyway. That's it for me.
+
+#### Other Topics
+
+[Onkar] Onkar needs a review on pull request. I think we can take that offline.
+
+[Christine] How does someone become a maintainer for MD and Web Talks?
+
+[Pranshu] Good question.
+
+[Ruth] It's usually, someone who has around for a long time which is helping with poor request reviews and issue triaging and those kind of activities and then we take it on a case-by-case basis.
+
+[Pranshu] We don't have a fixed guidelines yet. But I will be building those. So I will have a clear response for you in a few weeks, but, right now, just get involved, just get, get your hands dirty with issues, PRs, just help us out. Just, just like Onkar does here. Chris has here and a lot of good folks who are on the yeah, and After some time, I'll just add you to becoming a maintainer, honestly. It's as simple as that right now. But, I will get back to you with concrete guidelines. I'll reach out to you. And, yeah, we'll take it offline. No worries.
+
+[Onkar] I'm being asked to convert an image to a SVG here, but I'm not well-versed with SVG, how to proceed? Can we have a tool hosted somewhere to convert P and G to SVG for other contributors?
+
+[Brian] I can help you with that one Onkar. If you want, I've been doing a lot of the HTTP diagrams in the last couple of weeks. Mostly this has been redoing them from scratch, which has been okay if they're mermaids, then we have the Source for this afterwards. And otherwise, they're just boxes and lines. And this is pretty easy to recreate so I can give you a hand with that if you want.
+
+[Ruth] Thanks, Brian.
```

Comment:
```suggestion
[Onkar] I'm being asked to convert an image to an SVG, but I'm not well-versed with SVG. How should I proceed? Can we have a tool hosted somewhere to convert PNG to SVG for other contributors?

[Brian] I can help you with that, Onkar. I've been doing a lot of the HTTP diagrams in the last couple of weeks, mostly redoing them from scratch, which has been okay if they're mermaid diagrams, because then we have the source for them afterwards. Otherwise, if they're just boxes and lines, they're pretty easy to recreate, so I can give you a hand with that if you want.
```